src/lib/journey/journey.store.ts

```diff
@@ -142,6 +146,10 @@ export function initialize(initOptions?: StepOptions): JourneyStore {
       successful: false,
       response: null,
       recaptchaAction: nextOptions?.recaptchaAction,
+      pingProtect: {
+        envId: nextOptions?.pingProtect?.envId ?? '',
+        consoleLogEnabled: nextOptions?.pingProtect?.consoleLogEnabled,
+      },
```

Comment:
Can you elaborate on why you added this data to the store's value object?

Reply:
i was following the same basic flow as recaptcha. I need these options to pass through to the generic component at some point. What is the alternative?

Reply:
this is backed out